plugins/azure/resoto_plugin_azure/resource/network.py

```diff
@@ -4266,8 +4231,8 @@ def connect_in_graph(self, builder: GraphBuilder, source: Json) -> None:
             path="/subscriptions/{subscriptionId}/providers/Microsoft.Network/networkInterfaces",
             path_parameters=["subscriptionId"],
             query_parameters=["api-version"],
-            compared_property=compared_property,
-            binding_property=lambda r: r["properties"]["virtualMachine"]["id"]
+            first_property=first_property,
+            second_property=lambda r: r["properties"]["virtualMachine"]["id"]
             if "properties" in r and "virtualMachine" in r["properties"]
             else "",
         )
```

Comment:
This call is not required - it fetches all network interfaces, which have been already fetched and are available in the graph.